### DISCUSSION TITLE
Bundle Command

### DISCUSSION THREAD

**Discussion by ad-chuliang** [wixtoolset/issues#8657] [Questions] [answered]
Msi file has msiexec commands. Now, I have a bootstrapper application. So, what commands that a bootstrapper application has? I have try command like "install.exe /quiet" to realize silent install. But, I need to Put together a document of run commands。

**Comment by bevanweiss** [comment, marked as answer]
https://github.com/wixtoolset/wix/blob/main/src/api/burn/WixToolset.BootstrapperApplicationApi/BootstrapperCommand.cs
